open-craft/edx-platform
# fix: bump ora2 to fix uploads

## Description

Bumps ORA to prevent starting multiple concurrent file uploads.

## Supporting information

- https://tasks.opencraft.com/browse/BB-9210

## Testing instructions

1. Deploy this branch to a devstack.
1. Go to a course containing an ORA with file uploads.
1. Verify the upload button is disabled when an upload starts and a spinner shows it's in progress.

## Deadline

None

## Other information

- Does this change depend on other changes elsewhere?
- Any special concerns or limitations? For example: deprecations, migrations, security, or accessibility.
